app/helpers/cache_helper.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module CacheHelper
+  # Override ActionView `cache` helper, adding the current locale to the cache
+  # key.
+  def cache(key = {}, options = {}, &block)
+    super([key, locale: I18n.locale], options, &block)
```

Comment:
Something this makes me think of: admin.

- We're caching in admin
- Admin pages shouldn't be translated (at least for the time being)
- Your `I18n` locale should always be english for admin pages (at least for the time being)

Reply:
That was my assumption -- I'm not working on the admin templates.

Reply:
Fair. The only reason this is pertinent to this PR is with this caching addition, if someone is in admin but has a different language header set, it will not use the correct cache.

Reply:
I'm foreseeing some (potential) duplicated caching (in practice not so much), am I missing something else though? 🤔 

In particular: If I have Dutch set as my locale and then switch to admin, cached admin HTML fragments will be duplicated under `locale/nl/`.

Reply:
No, this is the only thing. It just seems like something that would be easy to avoid by checking if `controller_namespace == "admin"`

`controller_namespace` is defined in [controller_helpers.rb](https://github.com/bikeindex/bike_index/blob/master/app/controllers/concerns/controller_helpers.rb#L83)